benchmarks/src/generated/main/grpc/io/grpc/benchmarks/proto/BenchmarkServiceGrpc.java

```diff
@@ -157,6 +157,12 @@ protected BenchmarkServiceFutureStub build(io.grpc.Channel channel,
     }
   }
 
+  public static abstract class AbstractBenchmarkService implements BenchmarkService, io.grpc.stub.BindableService {
```

Comment:
I see what you're saying. Are `BaseRouteGuideService` classes that you're using at Google providing something else besides just implementing service interface and returning `UNIMPLEMENTED` by default? Are they adding some base functionality shared across service methods? Would implementing those methods in new `AbstractRouteGuideService` eliminate the need of having `BaseRouteGuideService`?

I must say I quickly skimmed through #1469 at first and didn't fully understand it. I see the point of having "implemented-unimplemented" base classes now and I think we should implement this. I'll add this to my PR.